addons/mail/tests/test_mail_render.py

```diff
@@ -162,32 +134,33 @@ def test_evaluation_context(self):
             'custom_value': 'Custom Render Value'
         }
         srces = [
-            '<b>I am ${user.name}</b>',
-            '<span>Datetime is ${format_datetime(datetime.datetime(2021, 6, 1), dt_format="MM - d - YYY")}</span>',
-            '<span>Context ${ctx.get("custom_ctx")}, value ${custom_value}</span>',
+            '<b>I am {{ user.name }}</b>',
+            '<span>Datetime is {{ format_datetime(datetime.datetime(2021, 6, 1), dt_format="MM - d - YYY") }}</span>',
+            '<span>Context {{ ctx.get("custom_ctx") }}, value {{ custom_value }}</span>',
         ]
         results = [
             '<b>I am %s</b>' % self.env.user.name,
             '<span>Datetime is 06 - 1 - 2021</span>',
             '<span>Context Custom Context Value, value Custom Render Value</span>'
         ]
         for src, expected in zip(srces, results):
-            for engine in ['jinja']:
+            for engine in ['inline_template']:
                 result = MailRenderMixin.with_context(**custom_ctx)._render_template(
                     src, partner._name, partner.ids,
                     engine=engine, add_context=add_context
                 )[partner.id]
                 self.assertEqual(expected, result)
 
     @users('employee')
-    def test_render_mail_template_jinja(self):
-        template = self.env['mail.template'].browse(self.test_template_jinja.ids)
+    def test_render_mail_template_inline_template(self):
+        template = self.env['mail.template'].browse(self.test_template_inline_template.ids)
         partner = self.env['res.partner'].browse(self.render_object.ids)
         for fname, expected in zip(['subject', 'body_html'], self.base_rendered):
             rendered = template._render_field(
                 fname,
                 partner.ids,
-                compute_lang=True
+                engine='qweb',
```

Comment:
That's not correct if I understood everything. All fields do not use qweb.

To propagate to other changes probably.